pages/Footer.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import Link from 'next/link';
+
+const Footer = () =>(
```

Comment:
Please move this to `src` directory as it is not a page but rather a shared component